features/step_definitions/basic_step.rb

```diff
@@ -0,0 +1,37 @@
+Then(/^show me the page$/) do
+  save_and_open_page
+end
+
+When(/^I visit "([^"]*)" page$/) do |page|
```

Comment:
Also, I would expect this step definition to be a generic one too. Meaning we can reuse it to visit different pages and not just the `root_page`